ts/Stock/StockToolsBindings.ts

```diff
@@ -2089,7 +2089,7 @@ var stockToolsBindings: Record<string, Highcharts.NavigationBindingsOptionsObjec
                 // line
                 lastPrice: {
                     enabled: !lastPrice,
-                    color: 'red'
+                    color: (options.lastPrice && options.lastPrice.color) || '#ff0000'
```

Comment:
Is this necessary to define `color` here? It should be enough to toggle the `enabled` option from the button perspective, right?